### DISCUSSION TITLE
Having issues going back to SD1.5 after trying out SDXL

### DISCUSSION THREAD

**Discussion by Traslogan** [vladmandic/sdnext#1839] [Q&A]
Hey, I was trying out SDXL for a few minutes on the Vlad WebUI, then decided to go back to my old 1.5 stuff. I realized things looked worse, and the time to start generating an image is a bit higher now (an extra 1-2s delay). On top of this none of my existing metadata copies can produce the same output anymore. I tried undoing the stuff for the new pipeline and even restoring defaults for the webUI. After full restarts it's still producing completely different images than what I would expect to get. Anyone know what I might have done or what to flick on or off?

I've confirmed with others that they're still getting the normal intended images from when we both grab identical metadata images to copy on civitai, so I've no idea what I might have done to seemingly permanently screw up my image gen.

Edit: I know the WebUI downloaded something when I switched to all the SDXL stuff, was probably like 1GB or something, but Idk what it downloaded at that time. I've since completely deleted and reinstalled from scratch on a fresh git clone, and still having the issue that images are off.

**Comment by Tillerz** [comment]
Can you generate an image with 1.5 and then paste the output of the text data here? The stuff shown below the generated image? Maybe someone can find something that doesn't belong there. :)

**Comment by Seafan75** [comment]
I suggest that you go through the extensions (any if you had any lora/lycoris settings). I had problems with extensions being disabled when I switched back and it caused multiple hard to guess problems.
